### ISSUE TITLE
Make an effect that involves the use of Native UI Picker

### ISSUE THREAD

**Issue by lopeselio** [ACM-VIT/SparkAR-Creators#11]
This issue would need multiple commits(projects, resources) on ideas involving use of Native-UI Picker. Please provide a description with your idea and discuss for help with your project.

**Comment by distroinfinity** [comment]
Hey there!  I want to work on this issue but I am not a registered participants from Learn2Augment. Can I still contribute to this repo?

**Comment by rosy-02** [comment]
Yes surely. Learn2Augemnet is now over and the repository is open to the entire community. Will update the Readme soon.

**Comment by lopeselio** [comment]
@distroinfinity yes surely. The repository is open to all. 
1. Make sure you create a new issue if you want to contribute a new project.
2. If you want to contribute to this issue, make a pull request with your idea and provide description. 
3. You can also contribute resources for ongoing projects.

**Comment by distroinfinity** [comment]
@lopeselio I can make an effect(involving Native UI Picker) which closes this issue. In which directory should I paste the arexport file for the effect?

**Comment by lopeselio** [comment]
@distroinfinity What are you trying to make using the Native UI Picker feature. We would appreciate the project folder rether than the .arexport. Should involve a your assets and your .arproj file. Make sure the effect is less than 5 MB. If we are good to go, then you can create a new folder for your project with all the resouces and the .arproj file. The projects is to be added in the `Projects/Native-UI-Picker` directory

**Comment by distroinfinity** [comment]
@lopeselio I am thinking of creating a simple effect in which you can try 3D glasses(one at a time) and adding a Native Ui Picker to facilitate the user to try multiple glasses from the options made available through UI picker, 3-4 options should be suffice in my opinion and as far as the 3D models for glasses are concerned, there are many options available on Spark AR library so it should not be am issue. Let  me know what you think of it

**Comment by lopeselio** [comment]
@distroinfinity Cool that's a great idea. Create pull request in the `Projects/Native-UI-Picker`. Will it suffice the size requirement if you are using 3D assets with A Native UI-Picker. If it does, then let's go ahead

**Comment by lopeselio** [comment]
@distroinfinity This is a great idea. Will be really helpful

**Comment by distroinfinity** [comment]
@lopeselio Yes I am working on it.

**Comment by lopeselio** [comment]
Merged PR https://github.com/ACM-VIT/SparkAR-Creators/pull/16 #16 from @distroinfinity. More projects invited.

**Comment by saumy007** [comment]
Is this project still open 
I can add native UI picker
